File: redvelvet/src/main/java/com/redvelvet/business/ingredient/entity/model/Ingredient.java
package com.redvelvet.business.ingredient.entity.model;

import com.redvelvet.business.base.entity.BaseEntity;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;

@Entity
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Ingredient extends BaseEntity {

    @Column(nullable = false)
    String name;

}

File: redvelvet/src/main/java/com/redvelvet/business/ingredient/entity/model/Ingredient.kt
package com.redvelvet.business.ingredient.entity.model

import com.redvelvet.business.base.entity.BaseEntity
import jakarta.persistence.Column
import jakarta.persistence.Entity
import org.jetbrains.annotations.NotNull

@Entity
class Ingredient : BaseEntity() {

    @Column(nullable = false)
    var name: String? = null

    @Column(nullable = false)
    var approved: Boolean = false
}
